lib/cocoapods/installer/target_installer/aggregate_target_installer.rb

```diff
@@ -118,7 +120,7 @@ def create_copy_resources_script
         target.user_build_configurations.keys.each do |config|
           file_accessors = library_targets.select { |t| t.include_in_build_config?(config) }.flat_map(&:file_accessors)
           resource_paths = file_accessors.flat_map { |accessor| accessor.resources.flat_map { |res| res.relative_path_from(project.path.dirname) } }
-          resource_bundles = file_accessors.flat_map { |accessor| accessor.resource_bundles.keys.map { |name| "${BUILT_PRODUCTS_DIR}/#{name}.bundle" } }
+          resource_bundles = file_accessors.flat_map { |accessor| accessor.resource_bundles.keys.map { |name| "${BUILT_PRODUCTS_DIR}/#{Shellwords.shellescape(name)}.bundle" } }
```

Comment:
So, instead of this change. Just changing the copy resources script to look something like:

```ruby
script += "  install_resource #{resource.shellescape}\n"
```